crates/modelardb_server/src/storage/uncompressed_data_buffer.rs

```diff
@@ -389,27 +403,29 @@ mod tests {
     }
 
     #[test]
-    fn test_check_if_in_memory_data_buffer_is_updated_by_batch() {
+    fn test_check_if_in_memory_data_buffer_is_unused() {
+        // CURRENT_BATCH_INDEX is purposely not used to make all of the batch indexes visible in the test.
```

Comment:
Very minor but maybe use "indices" instead of "indexes" since it is used elsewhere.

Reply:
Deleted the comment since it is no longer relevant as `CURRENT_BATCH_INDEX` is used in the test.